### PR TITLE
Latest Pr

### PR DESCRIPTION
**Pull request by numrahtbox**
Observe this carefully

### CODE REVIEW

File: NewFile.txt [hunk 1]
```diff
@@ -1,5 +1,4 @@
 Added
 new line
-
```
Comment (Numrahmubeen) [inline NewFile.txt:3]: Add A forth line

**Review by Numrahmubeen — CHANGES_REQUESTED** [review]

File: NewFile.txt [hunk 2]
```diff
@@ -1,5 +1,4 @@
 Added
 new line
```
Comment (numrahtbox) [inline NewFile.txt:2]: NO NEED OF IT RIGHT

File: NewFile.txt [hunk 3]
```diff
@@ -1,5 +1,4 @@
 Added
```
Comment (Numrahmubeen) [inline NewFile.txt:1]: 1ST COMMENT

Comment (Numrahmubeen) [inline NewFile.txt:1]: 2ND COMMENT

**Pull request merged**